man/src/oxide-bar.1.md

```diff
@@ -0,0 +1,51 @@
+% OXIDE-BAR(1) oxide-bar 0.1.0
+% Philipp Kalinowski
+% February 2023
+
+# NAME
+
+oxide-bar - statusbar for Oxide
+
+# DESCRIPTION
+
+Define the behavior of the statusbar for OxideWm. The config file provides the posibillity to customise the text and background color of the oxide statusbar.
```

Comment:
I guess we can name it just Oxide here, without the wm. Meanwhile this is the way we do it in the other documents :) But we should look up every document before the submission anyway.